nevergrad/instrumentation/multivariables.py

```diff
@@ -188,6 +188,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__()
         self.keywords: Tuple[Optional[str], ...] = ()
         self.probably_noisy = False  # True if for some reason we conjecture that the objective function is noisy.
+        self.scrambled = False  # True if there is no metric on the domain.
```

Comment:
is there any better name? maybe `non_measurable` or whatever is the term? that seems more precise that `scrambled` 

Reply:
i prefer scrambled as "non-measurable" means something in measure theory...
i put a better word as soon as I find

Reply:
ok : non-metrizable